Add docs page for bulk operations (#21056)
sourcegraph/sourcegraph: doc/batch_changes/how-tos/bulk_operations_on_changesets.md

# Bulk operations on changesets

> Sometimes it might be necessary to perform an action over many changesets in a batch change. Bulk operations provide a set of actions to run over a large set of them.

Reviewer: I wonder if we could combine this into one sentence:

```suggestion
Bulk operations allow a single action to be performed across many changesets in a batch change.
```

## Selecting changesets for a bulk operation

1. Click the checkbox next to a changeset in the list view. You can select all changesets you have permission to view.

    <img src="https://sourcegraphstatic.com/docs/images/batch_changes/select_changeset.png" class="screenshot">
1. If you like, select all changesets in the list by using the checkbox in the list header.
    
    <img src="https://sourcegraphstatic.com/docs/images/batch_changes/select_all_changesets_in_view.png" class="screenshot">
    

>     If you want to select _all_ changesets that meet the filters and search currently set, click the **(Select XX changesets) link in the header toolbar.

Reviewer: ```suggestion
    If you want to select _all_ changesets that meet the filters and search currently set, click the **(Select XX changesets)** link in the header toolbar.
```
